lib/active_merchant/billing/gateways/paysafe.rb

```diff
@@ -147,17 +145,20 @@ def add_address_for_vaulting(post, options)
 
       # This data is specific to creating a profile at the gateway's vault level
       def add_profile_data(post, payment, options)
-        address = options[:billing_address] || options[:address]
-
         post[:firstName] = payment.first_name
         post[:lastName] = payment.last_name
         post[:dateOfBirth] = {}
         post[:dateOfBirth][:year] = options[:date_of_birth][:year]
```

Comment:
I know this isn't exactly germane to your `phone` changes, but I'm wondering if we need to protect against scenarios where `options` doesn't have a `date_of_birth` object here?